### DISCUSSION TITLE
Can anyone make a video of some sort of how to make this work

### DISCUSSION THREAD

**Discussion by ghost** [gorilla-devs/ferium#286] [Q&A] [answered]
Personally i can barely even understand what im supposed to do, if there is already one please link it!

**Comment by theRookieCoder** [comment, marked as answer]
This program is a CLI, meaning that it has to be used in a command prompt or terminal. These aren't easy to use if you've never used one before, so I recommend you use something like [Prism Launcher](https://prismlauncher.org).

If you'd like to learn how to use CLIs [the readme](https://github.com/gorilla-devs/ferium#overview--help-page) has a tutorial-like section, I hope it helps.

**Reply by ghost** [reply]
I am new to it, do you know any non launcher options for auto updating mods (that are not discontinued)

**Reply by theRookieCoder** [reply]
[Prism Launcher](https://prismlauncher.org) can update mods too, and is actively maintained.

**Reply by ghost** [reply]
so ive decided to try and figure out how prism works and it works great! thanks for the help!
